File: abc/abc172/abc172_b.cs
// https://atcoder.jp/contests/abc172/submissions/14779393

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        var S = Console.ReadLine();
        var T = Console.ReadLine();

        Console.WriteLine(S.Zip(T, (s, t) => s != t).Count(b => b));
    }
}

File: abc/abc172/abc172_b.fs
// https://atcoder.jp/contests/abc172/submissions/14779160

let S = stdin.ReadLine().ToCharArray()
let T = stdin.ReadLine().ToCharArray()

Array.zip S T
|> Array.filter (fun (s, t) -> s <> t)
|> Array.length
|> stdout.WriteLine
